StressTest/StressTest/ViewController.swift

```diff
@@ -240,16 +240,16 @@ class ViewController: UIViewController {
             Exp(.linear)
             Exp(.zoom)
             0
-            UIColor.red.expression
+            UIColor.red
             14
-            UIColor.blue.expression
+            UIColor.blue
         }
 
         if let jsonObject = try? exp.jsonObject() {
-            color = mapView.__map.getStyleLayerProperty(forLayerId: land, property: "background-color")
+            color = try! mapView.__map.getStyleLayerProperty(forLayerId: land, property: "background-color")
```

Comment:
I'm a little concerned about these force trys. Even though its a test class, we should be catching these